rclpy/src/rclpy/_rclpy.c

```diff
@@ -2315,6 +2316,10 @@ rclpy_destroy_entity(PyObject * Py_UNUSED(self), PyObject * args)
       pyentity, "rcl_guard_condition_t");
     ret = rcl_guard_condition_fini(guard_condition);
     PyMem_Free(guard_condition);
+  } else if (PyCapsule_IsValid(pyentity, "rcl_event_t")) {
```

Comment:
Would you mind using `rclpy.handle.Handle` instead of adding to `rclpy_destroy_entity`? This method has been removed in #319.

Reply:
Yes, I am refactoring the code that uses capsules to use `Handle`